lib/rucio/core/rse.py

```diff
@@ -782,13 +795,32 @@ def parse_checksum_support_attribute(checksum_attribute):
 
     if not checksum_attribute:
         return GLOBALLY_SUPPORTED_CHECKSUMS
+
+    supported_checksum_list = checksum_attribute.split(',')
+
+    if 'none' in supported_checksum_list:
+        return []
     else:
-        supported_checksum_list = [c.strip() for c in checksum_attribute.split(',') if c.strip()]
-        if 'none' in supported_checksum_list:
-            return []
         return supported_checksum_list
 
 
+@read_session
+def get_rse_is_checksum_supported(checksum_name: str, rse_id: str = None, session=None) -> bool:
```

Comment:
this function isn't used anywhere. Why add it ? Do you intend to use it anywhere?